### DISCUSSION TITLE
GitHub Education apply rejected : my school isn't listed but other school form the same network are listed

### DISCUSSION THREAD

**Discussion by Justfey** [community/community#138556] [GitHub Education]
### Hi everyone,

Hi everyone, I am asking for your help, because I tried to enroll to the GitHub Education community, I send my student card, I have been rejected, I think it come from the fact that my school is at Cherbourg (ISPN) but GitHub purpose me only ISPN "Le Havre" and ISPN "Rouen" , what if I sent my scolarity certificate ? 

Thanks for your help, I really need GitHub Education for my study.

**Comment by alx-mp4** [comment]
Hi @Justfey, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
